play_motion:
  motions:
    head_look_down:
      joints: [torso_lift_joint, head_1_joint, head_2_joint, gripper_left_finger_joint, gripper_right_finger_joint]
      points:
      - positions: [0.08, 0.0, -0.4, 0.05, 0.05]
        time_from_start: 2.0

    open_gripper:
      joints: [gripper_left_finger_joint, gripper_right_finger_joint]
      points:
      - positions: [0.05, 0.05]
        time_from_start: 4.0

    lift_torso:
      joints: ['torso_lift_joint']
      points:
      - positions: [0.34]
        time_from_start: 2.5

    lower_head:
      joints: ['head_1_joint', 'head_2_joint']
      points:
      - positions: [0.0, -0.75]
        time_from_start: 2.0

    back_to_default:
      joints: [torso_lift_joint, head_1_joint, head_2_joint]
      points:
      - positions: [0.22, 0.0, 0.0]
        time_from_start: 2.0

    pick_final_pose:
      joints: [arm_1_joint,
      arm_2_joint, arm_3_joint, arm_4_joint, arm_5_joint,
      arm_6_joint, arm_7_joint]
      points:
      - positions: [0.21, -0.2, -2.2, 1.15, -1.57, 0.2, 0.0]
        time_from_start: 10.0

    pregrasp:
      joints: [arm_1_joint,
      arm_2_joint, arm_3_joint, arm_4_joint, arm_5_joint,
      arm_6_joint, arm_7_joint]
      points:
      - positions: [0.20, -1.34, -0.20, 1.94, -1.57, 1.37, 0.0]
        time_from_start: 0.0
      - positions: [0.20, -1.34, -0.20, 1.94, -1.57, 1.37, 0.0]
        time_from_start: 2.0
      - positions: [0.21, -1.02, -0.20, 1.94, -1.57, 1.52, 0.0]
        time_from_start: 3.5
      - positions: [0.21, 0.35, -0.2, 2.0, -1.57, 1.52, 0.0]
        time_from_start: 6.5
      - positions: [0.21, 0.35, -0.2, 0.0, -1.57, 1.52, 0.0]
        time_from_start: 10.0
      - positions: [0.21, 0.35, -3.0, 0.0, -1.57, 1.52, 0.0]
        time_from_start: 12.0
      - positions: [0.05, -0.07, -3.0, 1.5, -1.57, 0.2, 0.0]
        time_from_start: 17.0

    head_look_around:
      joints: [head_1_joint, head_2_joint]
      points:
      - positions: [0, 0]
        time_from_start: 0.1
      - positions: [0, -0.7]
        time_from_start: 3.0
      - positions: [-0.7, -0.7]
        time_from_start: 6.0
      - positions: [-0.7, -0.5]
        time_from_start: 9.0
      - positions: [0, -0.5]
        time_from_start: 12.0
      - positions: [0.7, -0.5]
        time_from_start: 15.0
      - positions: [0.7, -0.7]
        time_from_start: 18.0
      - positions: [0, -0.7]
        time_from_start: 21.0
      - positions: [0, -0.5]
        time_from_start: 24.0
